src/openrct2/interface/console.c

```diff
@@ -1217,6 +1217,21 @@ static sint32 cc_remove_unused_objects(const utf8 **argv, sint32 argc)
     return 0;
 }
 
+static sint32 cc_remove_park_fences(const utf8 **argv, sint32 argc)
+{
+    map_element_iterator it;
+    map_element_iterator_begin(&it);
+    do {
+        if (map_element_get_type(it.element) == MAP_ELEMENT_TYPE_SURFACE) {
+            // Remove all park fence flags
+            it.element->properties.surface.ownership &= 0xF0;
+        }
+    } while (map_element_iterator_next(&it));
```

Comment:
(also applies to this line)

Reply:
I think the while statement should be on the ending brace line as well, to be honest. Otherwise it makes while look like a separate command from the do statement.

Reply:
Our code style prescribes this. Discussions about their merit goes in the chat or an issue, but not in a PR.